packages/components/src/styles/form-styles.ts

```diff
@@ -1,35 +1,25 @@
 import type { JssStyle, Styles } from 'jss';
-import type { BreakpointCustomizable, Theme } from '../types';
-import { buildResponsiveStyles } from '../utils';
-import {
-  addImportantToRule,
-  getHiddenTextJssStyle,
-  getThemedColors,
-  getTransition,
-  hoverMediaQuery,
-  prefersColorSchemeDarkMediaQuery,
-} from './';
+import type { Theme } from '../types';
+import { getThemedColors, getTransition, hoverMediaQuery, prefersColorSchemeDarkMediaQuery } from './';
 import {
   borderRadiusSmall,
   borderWidthBase,
   fontLineHeight,
-  fontSizeTextXSmall,
+  spacingStaticMedium,
   spacingStaticSmall,
   spacingStaticXSmall,
   textSmallStyle,
 } from '@porsche-design-system/utilities-v2';
 import { getThemedFormStateColors } from './form-state-color-styles';
 import type { FormState } from '../utils/form/form-state';
 
-// TODO: should be removed if possible?
-export const INPUT_HEIGHT = 54;
-
 export type ChildSelector = 'input' | 'select' | 'textarea';
 
-export const getBaseChildStyles = (
+export const getSlottedTextFieldTextareaSelectStyles = (
   child: ChildSelector,
   state: FormState,
   theme: Theme,
+  isLoading: boolean,
```

Comment:
I think usually we tried to have `theme` as last parameter unless there are optional ones for consistency.